junit-platform-engine/src/main/java/org/junit/platform/engine/discovery/DiscoverySelectors.java

```diff
@@ -46,7 +46,8 @@
 @API(Experimental)
 public final class DiscoverySelectors {
 
-	private static Pattern fullyQualifiedMethodNamePattern = Pattern.compile("([^#]+)#([^(]+)(?:\\((.*)\\))?");
+	private static final Pattern FULLY_QUALIFIED_METHOD_NAME_PATTERN = Pattern.compile(
```

Comment:
This is good catch, converting this to an idiomatic constant. Is this just a refactoring or is this part of improving immutability? 

Thanks!
Giorgos